packages/components/src/components/badges/badge.tsx

```diff
@@ -0,0 +1,43 @@
+import { Component, Prop, h } from '@stencil/core';
+import { CssClassMap } from '../../utils/utils';
+import classNames from 'classnames';
+
+@Component({
+    tag: 't-badge',
+    styleUrl:
+        'badge.css',
+    shadow: true
+})
+
+export class Badge {
+    @Prop() size?: string = '';
+    @Prop() variant?: string = '';
+    @Prop() pill?: string = '';
+
+
+    private getCssClassMap(): CssClassMap {
+        return classNames(
+            'badge',
+            this.size && `badge--size-${this.size}`,
+            this.variant && `badge--variant-${this.variant}`,
+            this.pill && `badge--pill-${this.pill}`,
+        );
+    }
+
+    render() {
+        return (
+            <div class={this.getCssClassMap()}>
+                <div class="container">
+                    <span class="badge badge-pill badge--primary">Primary</span>
```

Comment:
Please return the html for the badge once, why would you need a `container` here?

Reply:
solved

Reply:
solved. also edit background for badge-default and badge-pill, add badge-link